tests/snuba/search/test_backend.py

```diff
@@ -472,6 +472,74 @@ def test_bookmarked_by_with_environment(self):
         )
         assert set(results) == set([])
 
+    def test_search_filter_query_with_custom_priority_tag(self):
+        priority = 'high'
+        self.store_event(
+            data={
+                'fingerprint': ['put-me-in-group2'],
+                'timestamp': (self.group2.first_seen + timedelta(days=1)).isoformat()[:19],
+                'stacktrace': {
+                    'frames': [{
+                        'module': 'group2'
+                    }]
+                },
+                'message': 'group2',
+                'tags': {
+                    'priority': priority,
+                },
+            },
+            project_id=self.project.id,
+        )
+
+        results = self.make_query(
+            search_filter_query='priority:%s' % priority,
+            tags={'priority': priority},
+        )
+
+        assert set(results) == set([self.group2])
+
+    def test_search_filter_query_with_custom_priority_tag_and_priority_sort(self):
+        priority = 'high'
+        for i in range(1, 10):
```

Comment:
I'd probably just make 2-3 events typically if that works for the sort, just to save on making the test slow